consensus/mint-client/src/config.rs

```diff
@@ -452,7 +485,10 @@ pub struct Config {
     pub command: Commands,
 }
 
-pub fn load_key_from_pem(filename: &str) -> Result<MintPrivateKey, String> {
+// a purpose-build pem loader for MintPrivateKey to avoid implementing
+// DistinguishedEncoding trait. MintPrivateKey was needed to implement Clone
+// trait for use with clap
```

Comment:
@jcape (not a blocking comment, just curious) - it looks like clap uses Clone in the vector construction, which forces us to add a "Clone" method to a private key (which, iiuc, is functionality the `Ed25519Private` explicitly omits for security reasons). Curious your thoughts there? (I don't think it's much of a security issue for this utility)